feedback/client/src/reviews_components/flag_components/modal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export function ModalDiv(props) {
```

Comment:
Since you're only using props.close, you can do something like 
`export function ModalDiv({ close })`
and use close directly for code clarity / cleanliness